### PR TITLE
Added gridlayout

### PR DESCRIPTION
**Pull request by ghinda**
Added gridlayout library.

https://ghinda.net/gridlayout/

### CODE REVIEW

File: files/gridlayout/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "gridlayout",
+  "repo": "ghinda/gridlayout",
+  "files": {
+    "include": ["gridlayout.css", "gridlayout.min.css", "gridlayout-ie.js", "gridlayout-ie.min.js"]
```
Comment (jimaek) [inline files/gridlayout/update.json]: Can you remove the unminified files from here? They are not required in production

**Comment by ghinda** [comment]
I removed the unminified files from the list of files in `update.json`.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by ghinda** [comment]
Thanks!

**Comment by jimaek** [comment]
:+1:

**Pull request closed without merging**